packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -151,9 +151,10 @@ export class BrowserCriClient {
         onAsynchronousError,
         onReconnect,
         protocolManager,
+        fullyManageTabs,
       })
 
-      const browserCriClient = new BrowserCriClient(browserClient, versionInfo, host!, port, browserName, onAsynchronousError, protocolManager)
+      const browserCriClient = new BrowserCriClient(browserClient, versionInfo, host!, port, browserName, onAsynchronousError, protocolManager, fullyManageTabs)
```

Comment:
There are quite a few parameters to this constructor now - should we refactor to a parameter object for better readability?

Reply:
Good call. This is somewhat existing code, but good time for a cleanup: [`0d5cad4` (#28105)](https://github.com/cypress-io/cypress/pull/28105/commits/0d5cad4fe2c7e55e625e54d410ba14aceed04f60)